packages/gestalt/src/boxTypes.js

```diff
@@ -0,0 +1,253 @@
+// @flow strict
+import PropTypes from 'prop-types';
```

Comment:
moved all Box Flow types here, and most proptypes as well. a central location should make future updates easier